src/gpu_sysman.c

```diff
@@ -331,35 +362,36 @@ static void log_uuid(const char *prefix, const uint8_t *byte, int len) {
   INFO("%s", buf);
 }
 
-/* Log Sysman API provided info for given GPU if logging is enabled
- * and on success, return GPU PCI ID as string in BDF notation:
- *   https://wiki.xen.org/wiki/Bus:Device.Function_(BDF)_Notation
+/* If GPU info setting is enabled, log Sysman API provided info for
+ * given GPU, and set PCI device ID to 'pci_dev'.  On success, return
```

Comment:
```suggestion
 * given GPU, and set PCI device ID to 'pci_dev'. On success, return
```

Reply:
It was intentional: https://en.wikipedia.org/wiki/Sentence_spacing#Computer_era